collectors/python.d.plugin/pandas/metadata.yaml

```diff
@@ -0,0 +1,310 @@
+# yamllint disable rule:line-length
+---
+meta:
+  plugin_name: python.d.plugin
+  module_name: pandas
+  monitored_instance:
+    name: Pandas
+    link: https://learn.netdata.cloud/docs/data-collection/generic-data-collection/structured-data-pandas
```

Comment:
this will be a 404 if we change it, but for now we could leave it, we don't use this link in the template atm